src/lib/seam/components/AccessCodeForm/TimezonePicker/TimezonePickerListItem.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useRef } from 'react'
+
+import { getTimezoneLabel, getTimezoneOffset } from 'lib/dates.js'
+import { CheckIcon } from 'lib/icons/Check.js'
+
+interface TimezoneListItemProps {
+  value: string
+  isSelected: boolean
+  onClick: () => void
+}
+
+export function TimezoneListItem({
+  value,
+  onClick,
+  isSelected,
+}: TimezoneListItemProps): JSX.Element {
+  const elRef = useRef<HTMLLIElement>(null)
+
+  useEffect(() => {
+    if (!isSelected) {
+      return
+    }
+
+    const { current: item } = elRef
+
+    if (item == null) {
+      return
+    }
+
+    const container = item.closest('.seam-timezones')
+
+    if (container == null) {
+      return
+    }
+
+    const { height: containerHeight } = container.getBoundingClientRect()
```

Comment:
This looks similar to the logic we use to position the filter box, but we have this bug https://github.com/seamapi/react/issues/259

Can we use native `<select>` instead?

Reply:
Sure, added `TimezonPicker` https://github.com/seamapi/react/pull/282/files#diff-66e36b728600047d9d211deedbdd8186d978821adc5bc075e46c35e55baa7068